sos/report/plugins/microshift.py

```diff
@@ -146,7 +147,13 @@ def setup(self):
         Output format for this function is based on `oc adm inspect` command,
         which is used to retrieve all API resources from the cluster.
         """
-        self.add_forbidden_path('/var/lib/microshift')
+        self.add_service_status('greenboot-healthcheck')
+        self.add_journal('greenboot-healthcheck')
+
+        if os.path.exists('/var/lib/microshift-backups'):
```

Comment:
Use Plugin's `self.path_exists()` here, as this is a smaller wrapper to account for when we're running in a container and need to collect from the host and not the container filesystem.

Reply:
Done.